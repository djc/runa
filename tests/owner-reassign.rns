def main():
	x = str(2014)
	print(x)
	x = str(2015)
	print(x)
	if True:
		x = str(2016)
		print(x)
