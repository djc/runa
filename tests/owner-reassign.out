2014
2015
2016
